src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -227,9 +248,9 @@ static void Main()
 ";
             var comp = CreateCompilationWithMscorlib(source, parseOptions: TestOptions.Regular.WithTuplesFeature());
             comp.VerifyDiagnostics(
-                // (8,18): error CS8208: The Deconstruct method for type 'C.Deconstruct(out int, int)' must have only out parameters.
+                // (8,9): error CS1615: Argument 2 may not be passed with the 'out' keyword
```

Comment:
The former error was a bit better. User is not passing anything as "out".  
Or just report that no appropriate Deconstruct was found.